### ISSUE TITLE
Move to next screen when on Home page

### ISSUE THREAD

**Issue by akell** [cmauri/eva_facial_mouse#2]
When on the home screen we could onto find a way to move between screens to get to more apps.

Is there a way to swipe between screens?

Device: Samsung GII Model GT-9100. Android v 4.1.2

**Comment by cmauri** [comment]
Yes. Scroll buttons (see icon [1]) are intended for such a purpose. However, we have noticed that for most Samsung devices, these buttons do not show up in the default (Samsung) launcher. 

As a workaround you could try installing another launcher such as [2]. Once installed, just click the home screen background to scroll (when there is more than one option, a contextual menu will appear to choose between forward and backward scrolling).

These issues are due to the fact that the Android Accessibility API poses some restrictions on the actions that can be executed (e.g. swipe gesture).

[1] https://raw.githubusercontent.com/cmauri/eva_facial_mouse/master/eviacam/src/main/res/drawable-xhdpi/scrollfor_icon.png
[2] https://play.google.com/store/apps/details?id=com.google.android.launcher&hl=es

**Comment by akell** [comment]
OK installed the launcher and this works too.
Maybe obvious but 1) Remember to enable context menus as the scroll option will not appear and 2) Leave some screen real estate to click on otherwise you can end up just clicking on icons
